cvts/settings.py

```diff
@@ -116,6 +116,8 @@ def _get_path(var, must_exist=False):
 #: Path to Valhalla configuration file.
 VALHALLA_CONFIG_FILE = os.path.join(CONFIG_PATH, 'valhalla.json')
 
+DATA_LAKE = '/mnt/data_lake/'
```

Comment:
Can we change that to be configured from an environment variable in a similar way to other settings in this file?

Reply:
We can, but where are we setting up the variables?

Reply:
I do it in my .bash_local... not ideal and I have always intended to put a config file somewhere 'at the system level'